src/index.ts

```diff
@@ -1,22 +1,36 @@
-import path from 'path';
 import express from 'express';
-import { userRoutes, groupRoutes, userGroupRoutes } from './routers';
+import { userRoutes, groupRoutes, userGroupRoutes, authRoutes } from './routers';
 import { userLogger, groupLogger, logInfo, logError } from './utils/logger';
+import { checkToken } from './routers/controllers/authControllers';
+import cors from 'cors';
 
 const PORT = process.env.SERVER_PORT;
 
 const app = express();
+const corsOptions = {
+  methods: ['GET', 'PUT', 'POST', 'DELETE', 'OPTIONS']
+}
 
-// Configure Express to use EJS
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'ejs');
+app.use(cors(corsOptions));
+// либо можно вот так настроить корсы:
+// app.use((req, res, next) => {
+//   res.header('Access-Control-Allow-Origin', '*');
+//   res.header('Access-Control-Allow-Credentials', 'true');
+//   res.header('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE,OPTIONS');
+//   res.header(
+//     'Access-Control-Allow-Headers',
+//     'callid, authorization, content-type, x-finger-print, X-Device-FingerPrint'
+//   );
+//   next();
+// });
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
-app.use('/users', userRoutes, logInfo(userLogger), logError(userLogger));
-app.use('/groups', groupRoutes, logInfo(groupLogger), logError(groupLogger));
-app.use('/connections', userGroupRoutes);
+app.use('/users', checkToken, userRoutes, logInfo(userLogger), logError(userLogger));
+app.use('/groups', checkToken, groupRoutes, logInfo(groupLogger), logError(groupLogger));
+app.use('/connections', checkToken, userGroupRoutes);
```

Comment:
you can try to add to all routes `checkToken` by default, except authorization route

Reply:
Done 👍